src-tauri/src/commands.rs

```diff
@@ -390,6 +391,7 @@ pub async fn update_instance(
         }
         transaction.commit().await?;
         info!("Instance {} updated", instance_id);
+        app_handle.emit_all("instance-update", ())?;
```

Comment:
Nie wiem jak ten event jest potem wykorzystywany, ale nie przydałoby się coś więcej jako payload? ID może przynajmniej jak już jest pod ręką.

Reply:
This only tells to queryClient to invalidate instances query and that fetches all of them so it doesn't need any data bcs it will be unused anyway.